optimus/platform/projects/platform/src/main/scala/optimus/platform/runtime/RuntimeComponents.scala

```diff
@@ -61,9 +61,9 @@ object RuntimeComponents {
     DALEntityResolver(dsi)
 
   /** Extracts [[InitialRuntime.initialTime]] from a scenario stack */
-  private[optimus] def getInitialRuntimeInitialTime(ss: ScenarioStack): Instant = {
+  private[optimus] def getInitialRuntimeInitialTime(ss: ScenarioStack): Instant = ??? /* {
     ss.getNode(InitialRuntime.initialTime.key).get
-  }
+  } */
```

Comment:
Here and in other places I got compiler errors that I don't know how to silence:
```
[error] Optimus: (20515) Incorrect usage of optimus.graph.Node.get in optimus.platform.runtime.RuntimeComponents.getInitialRuntimeInitialTime. You probably should not be using it at all.
[error]     ss.getNode(InitialRuntime.initialTime.key).get
```